extensions/wikia/PhalanxII/classes/PhalanxStatsPager.class.php

```diff
@@ -78,4 +78,30 @@ function formatRow( $row ) {
 
 		return $html;
 	}
+
+	/**
+	 * Get connection to Phalanx Stats table on specials DB
+	 *
+	 * Phalanx stats table is encoded in utf-8, while in most cases MW communicates with
+	 * databases using latin1, so sometimes we get strings in wrong encoding.
+	 * The only way to force utf-8 communication (adding SET NAMES utf8) is setting
+	 * global variable wgDBmysql5.
+	 *
+	 * @see https://github.com/Wikia/app/blob/dev/includes/db/DatabaseMysqlBase.php#L113
+	 *
+	 * @param int $dbType master or slave
+	 * @return DatabaseBase
+	 */
+	protected function getDatabase( int $dbType = DB_SLAVE ): DatabaseBase {
+		$wrapper = new Wikia\Util\GlobalStateWrapper( [
+			'wgDBmysql5' => true
+		] );
+
+		$db = $wrapper->wrap( function () use ( $dbType ) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$db". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)